pootle/apps/pootle_store/models.py

```diff
@@ -842,8 +858,6 @@ def resurrect(self, is_fuzzy=False):
         self._save_action = UNIT_RESURRECTED
```

Comment:
they are kept for now - im trying to factor them out - so these flags can still be significant atm